apps/bmd/src/pages/PrintPage.tsx

```diff
@@ -31,14 +45,30 @@ const PrintPage: React.FC = () => {
       await printer.print()
       updateTally()
       printerTimer.current = window.setTimeout(() => {
-        resetBallot()
-      }, printerMessageTimeoutSeconds * 1000)
+        setIsPrinted(true)
+        window.setTimeout(
+          resetBallot,
+          verifyAndCastMessageTimeoutSeconds * 1000
+        )
```

Comment:
TODO: Handle cleanup of this on unmount.

Reply:
done.